build_tools/circle/build_wheels_arm.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -e
+set -x
+
+sudo add-apt-repository --remove ppa:ubuntu-toolchain-r/test
+sudo apt-get update
+sudo apt-get install python3-virtualenv ccache
+python3 -m virtualenv --system-site-packages --python=python3 testenv
+source testenv/bin/activate
+pip install --upgrade pip
+for pyversion in 37 38 39; do
+    pywheel=`echo "cp"$pyversion"-manylinux_aarch64"`
+    export CIBW_BUILD=$pywheel
```

Comment:
We also need to configure cibuildwheel to run the scikit-learn tests on the generated wheel by defining the `CIBW_TEST_REQUIRES` and `CIBW_TEST_COMMAND` variables as done in:

https://github.com/scikit-learn/scikit-learn/blob/61ad2657a86dba20e15e4d76f58322d9c36ecc9e/.github/workflows/wheels.yml#L102-L103

Reply:
done